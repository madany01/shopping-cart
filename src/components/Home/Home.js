import { Link } from 'react-router-dom'
import homeImg from './home-img.jpg'

function Home() {
  return (
    <div className='home'>
      <div className='website-info'>
        <h1 className='large'>WebDev Courses</h1>
        <p className='md'>Learn Web development</p>
        <Link to='/products' className='products-btn'>
          Show Courses
        </Link>
      </div>
      <div className='img-wrap'>
        <img src={homeImg} alt='programmer typing in front of three screens' />
      </div>
    </div>
  )
}

export default Home
