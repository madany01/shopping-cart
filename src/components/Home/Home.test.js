import { render } from '@testing-library/react'

import Home from './Home'

jest.mock('react-router-dom', () => ({
  Link: ({ to, children, ...props }) => (
    <a href={to} {...props}>
      {children}
    </a>
  ),
}))

describe('Home', () => {
  it('matches the snapshot', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })
})
