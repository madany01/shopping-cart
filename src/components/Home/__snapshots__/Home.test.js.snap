// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches the snapshot 1`] = `
<div>
  <div
    class="home"
  >
    <div
      class="website-info"
    >
      <h1
        class="large"
      >
        WebDev Courses
      </h1>
      <p
        class="md"
      >
        Learn Web development
      </p>
      <a
        class="products-btn"
        href="/products"
      >
        Show Courses
      </a>
    </div>
    <div
      class="img-wrap"
    >
      <img
        alt="programmer typing in front of three screens"
        src="home-img.jpg"
      />
    </div>
  </div>
</div>
`;
